### PR TITLE
Contact form Validation

### PR DESCRIPTION
**Pull request by AlfiyaSiddique**
## Related Issue
Added Real-time validation in the contact form and prevent it from sending invalid data.

Closes: #223 **Real-time-validation in Contact.**

## Description of Changes
 1. I have created a contact form validation object in validate.js for validating the different fields.
 2. I have used react states to display the error messages below the respective fields.
 3.  I have checked for any errors in the submit form function, if the errors exist the function will not run submit form code and gives an alert. 
-->

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Sample Output

https://github.com/GrabBits/GrabBits_Website/assets/86224794/8158739f-377e-45b7-9670-c60e97c6462f

### CODE REVIEW

File: client/src/helper/validate.js [hunk 1]
```diff
@@ -94,4 +94,42 @@
     }
 
     return error;
-}
+}
+
+// Contact Form Validation
+
+const emailRegex = /^\w+([\.-]?\w+)*@(gmail\.com|yahoo\.com|hotmail\.com|aol\.com|outlook\.com)$/;
```
[the remark this answers is not in the document]
Comment (Grab-Bits) [reply client/src/helper/validate.js:101]: Hey @AlfiyaSiddique , can you please look into this , also can we have this validations in the front form also!!
image

[the remark this answers is not in the document]
Comment (AlfiyaSiddique) [reply client/src/helper/validate.js:101]: Sure @Grab-Bits
I will add the validation in the front form as well and I'll fix the above problem.👍🏻

**Comment by AlfiyaSiddique** [comment]
Hello @Grab-Bits 

Resolve the above problem and added validation in the home page's contact form as well.

**Comment by AlfiyaSiddique** [comment]
Hello @Grab-Bits  please review this PR!

**Comment by Grab-Bits** [comment]
Great work @AlfiyaSiddique  , Merging this !!

**Comment by Bhushan8177** [comment]
Thank you for Contributing to GrabBit!!
Please ⭐ the repository and follow me on Github ...!

You can connect with me on Linkedin - 
https://www.linkedin.com/in/bhushan-kale/

**Comment by AlfiyaSiddique** [comment]
@Bhushan8177  or @Grab-Bits 
Can you please add gssoc23 labels to this pr

**Comment by Bhushan8177** [comment]
Done @AlfiyaSiddique !

**Comment by AlfiyaSiddique** [comment]
> Done @AlfiyaSiddique !

Thanks a lot 👍

**Pull request merged**